javascript/computer_science/recursive_methods.md

```diff
@@ -25,7 +25,7 @@ This section contains a general overview of topics that you will learn in this l
 
 <div class="lesson-content__panel" markdown="1">
   1. Read [this recursion resource](https://javascript.info/recursion) for a good intro to recursion.
-  2. Watch [this explanation of Recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified.
+  2. Watch [this explanation of Recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified and [this explanation of Recursion](https://youtu.be/LteNqj4DFD8) by DevSage.
```

Comment:
In watching both of these videos, I think the first example of the DevSage one is overkill. However, the second one has a great explanation. Therefore I would like to add a timestamp to this video (and remove the capitalization of Recursion). I was thinking something like this:

```suggestion
  2. Watch [this explanation of recursion](https://www.youtube.com/watch?v=6oDQaB2one8) by Web Dev Simplified and [this additional example of recursion](https://youtu.be/LteNqj4DFD8?t=340) by DevSage.
```